### ISSUE TITLE
Conflict between `Layout/ArgumentAlignment` and `Layout/FirstHashElementIndentation`

### ISSUE THREAD

**Issue by natematykiewicz** [rubocop/rubocop#9829]
Given the config:

```yml
Layout/ArgumentAlignment:
  EnforcedStyle: with_first_argument # (default)
Layout/FirstHashElementIndentation:
  EnforcedStyle: consistent # (non-default)
```

With this code:

```ruby
it do
  is_expected.to contain_exactly(
    {
      a: 1, # <-- Layout/ArgumentAlignment
      b: 2, # <-- Layout/ArgumentAlignment
      c: 3, # <-- Layout/ArgumentAlignment
    },
    {
      a: 2,
      b: 3,
      c: 4,
    }
  )
end
```

`Layout/ArgumentAlignment` offenses are registered on the 3 indicated lines.

It seems that `Layout/ArgumentAlignment` wants this:

```ruby
it do
  is_expected.to contain_exactly(
    {
    a: 1, # <-- Layout/FirstHashElementIndentation
    b: 2,
    c: 3,
    },
    {
      a: 2,
      b: 3,
      c: 4,
    }
  )
end
```

This looks odd, but also then causes a violation on `Layout/FirstHashElementIndentation`.

`Layout/FirstHashElementIndentation` seems to want this:

```ruby
it do
  is_expected.to contain_exactly(
    {
      a: 1, # <-- Layout/ArgumentAlignment
    b: 2,
    c: 3,
    },
    {
      a: 2,
      b: 3,
      c: 4,
    }
  )
end
```

--------

## Expected behavior

This should be acceptable:

```ruby
it do
  is_expected.to contain_exactly(
    {
      a: 1,
      b: 2,
      c: 3,
    },
    {
      a: 2,
      b: 3,
      c: 4,
    }
  )
end
```

## Actual behavior

Rubocop is impossible to please and hits an infinite loop attempting to autocorrect.

```
Infinite loop detected in app/controllers/graphql_controller.rb and caused by Layout/FirstHashElementIndentation -> Layout/ArgumentAlignment
```

## Steps to reproduce the problem

Run Rubocop using the config and code specified above.

## RuboCop version

```
$ bundle exec rubocop -V
1.15.0 (using Parser 3.0.1.1, rubocop-ast 1.5.0, running on ruby 2.5.1 x86_64-linux)
  - rubocop-performance 1.11.3
  - rubocop-rails 2.10.1
  - rubocop-rspec 2.3.0
```

**Comment by natematykiewicz** [comment]
Is this possibly the fix, and I just need to wait? https://github.com/rubocop/rubocop/pull/9805
Note: I'm using `Layout/FirstHashElementIndentation` of `EnforcedStyle: consistent` where #9805 mentions `EnforcedStyle: with_fixed_indentation`. But maybe that's still the same problem.

**Comment by koic** [comment]
> Is this possibly the fix, and I just need to wait? #9805

https://github.com/rubocop/rubocop/pull/9812 will also be required to resolve this issue. Please wait for the next release as it has been fixed in master branch. Thank you.

**Comment by natematykiewicz** [comment]
Excellent. Thanks @koic!
